modules/cluster/main.tf

```diff
@@ -20,6 +23,7 @@ resource "null_resource" "create_cluster" {
     environment = {
       LOCATION       = var.location_name
       cluster_name   = var.cluster_name
+      host_zones     = self.triggers.host_zones
```

Comment:
Based on the pattern in place I think this should be `var.host_zones` instead to make things consistent with the other parameters.